routes/messages/commands/get_started.js

```diff
@@ -38,6 +39,11 @@ module.exports = async event => {
               title: book.title,
               image_url: book.cover_img,
               subtitle: `by ${book.author}`,
+              default_action: {
+                type: "web_url",
+                url: "https://cdn1.imggmi.com/uploads/2019/8/22/76c10c3d1b579bf0a66cb7f1cfe74843-full.jpg",
+                webview_height_ratio: "tall",
+              },
```

Comment:
This is a good idea, give the message some sort of functionality when clicked.  We currently don't do much of this

Reply:
This good to merge now?